doc/man3/OPENSSL_gmtime.pod

```diff
@@ -0,0 +1,62 @@
+=pod
+
+=head1 NAME
+
+OPENSSL_gmtime,
+OPENSSL_gmtime_adj,
+OPENSSL_gmtime_diff - platform-agnostic OpenSSL time routines
+
+=head1 SYNOPSIS
+
+ #include <openssl/crypto.h>
+
+ struct tm *OPENSSL_gmtime(const time_t *timer, struct tm *result);
+ int OPENSSL_gmtime_adj(struct tm *tm, int offset_day, long offset_sec);
+ int OPENSSL_gmtime_diff(int *pday, int *psec,
+                        const struct tm *from, const struct tm *to);
+
+=head1 DESCRIPTION
+
+OPENSSL_gmtime() returns the UTC time specified by B<timer> into the provided
+B<result> argument.
+
+OPENSSL_gmtime_adj() adds the offsets in B<offset_day> and B<offset_sec> to B<tm>.
+
+OPENSSL_gmtime_diff() calculates the difference between B<from> and B<to>.
+
+=head1 NOTES
+
+It is an error to call OPENSSL_gmtime() with B<result> equal to NULL. The
```

Comment:
Please use `I<>` when referencing the parameters instead of bold everywhere in the manpage.